scenes/enumerated/problem_12649/problem_12649.py

```diff
@@ -0,0 +1,448 @@
+from manim import *
+from hanrahashiv import *
+from qarakusiscene import TaskNumberBox
+
+
+FONT_SIZE=65
+SCALE_FACTOR = 1.5
+WIGGLE_SCALE_FACTOR = 1.25
+
+class Problem12649(FormulaModificationsScene):
+    def construct(self):
+        # Խնդրի համարը
+        taskNumberString=r"\textrm{Խ.} 12649"
+        taskNumber = TaskNumberBox(taskNumberString)
+        self.add(taskNumber)
+        self.wait(0.25)
+
+        # 4 վարժությունները
+        self.formulas = formulas = VGroup(
+            Tex('$2$', '$x$', r'$\cdot$', '$($', '$3$', '$x$', '$^2$', ' $+$ ',  '$1$', '$)$', font_size=FONT_SIZE).next_to(2*UP + 6.5*LEFT, buff=0.2),
+            Tex(r'$\frac{5}{3}$', '$x$', r'$\cdot$', '$($', '$6$', '$x$', '$^3$', ' $+$ ', '$2$', '$x$', '$y$', '$^2$', '$)$', font_size=FONT_SIZE).next_to(1*UP + 6.5*LEFT, buff=0.2),
+            Tex('$a$', '$^2$', '$b$', '$^3$', r'$\cdot$', '$($', '$2$', '$a$', '$b$', '$c$', ' $+$ ', '$b$', '$^2$', '$)$', font_size=FONT_SIZE).next_to(6.5*LEFT, buff=0.2),
+            Tex('$2$', '$x$', '$y$', r'$\cdot$', r'$($', '$($', '$x$', '$^2$', '$y$', '$)$', '$^3$', ' $+$ ', '$3$', '$x$', '$y$', '$)$', font_size=FONT_SIZE).next_to(DOWN + 6.5*LEFT, buff=0.2))
```

Comment:
Do you mind adding small comments on each line to make it more readable?
Like `#  (3x^2+1)`